### ISSUE TITLE
[Feature Request] Accessing sequences in arbitrary ranges via Genome class

### ISSUE THREAD

**Issue by gokceneraslan** [openvax/pyensembl#236]
It would be great to have a generic `.get_sequence(self, contig, position, end, strand)` function in the Genome class which extracts sequence from the genome given a range. This is similar to [getSeq function](https://www.rdocumentation.org/packages/BSgenome/versions/1.40.1/topics/getSeq-methods) in BSgenome package in R:

```R
getSeq(Celegans, 'chrI',  start=100, end=200, strand=+)
```

`.transcript_sequence()` and `.protein_sequence()` can also use this function.

Right now, it's not possible to get the full sequence of a gene including UTRs and introns. That'd make it easier for example.

**Comment by iskandr** [comment]
Closing as duplicate of #337 — once optional chromosome FASTA support lands, this becomes a one-liner against `genome.chromosome_sequences`. Subscribe to #337 for progress.
